### DISCUSSION TITLE
Allow remote access to Elasticsearch API in 2.4

### DISCUSSION THREAD

**Discussion by ConRock5000** [Security-Onion-Solutions/securityonion#11307] [2.4] [answered]
Hi,

In 2.4 so-allow points me to the SOC but I can not find a way to open port 9200 in there for Elasticsearch.

Is this the command I should be using from the console?
sudo /sbin/so-firewall includehost elasticsearch_rest 192.168.1.249
It accepts the command but the connection still does not work.

I can perform curl commands successfully on the local host but not remotely.
(I managed to enable firewalld and add the port myself on one build just to make sure I could access the Elasticsearch API remotely and there wasn't a different issue but I broke other things in that process and rebuilt.)

I originally thought the issue was do to configuring the AirGap setup, but now I see I can't figure this out no matter which way I set up Security Onion 2.4.

**Comment by ConRock5000** [comment]
Also, in the command line.
Trying
so-firewall addhostgroup elasticsearch
Just results in the usage comments being printed. It does this for every so-firewall command I try.

Update:
I can't add new groups, but I did figure out how to edit a portgroup.
It accepts the command but the connection still does not work.
sudo /sbin/so-firewall includehost elasticsearch_rest 192.168.1.249

**Comment by ConRock5000** [comment]
Back to AirGap install.
I can access the SOC from another machine on the network but I don't know how to allow access to Elasticsearch to perform curl commands.

**Comment by dougburks** [comment, marked as answer]
Instead of using the command-line, please try the web interface:
- go to the new Configuration interface
- at the top of the page, click the Options menu and then enable the `Show all configurable settings, including advanced settings.` option
- navigate to firewall --> hostgroups --> elasticsearch_rest

![Screenshot 2023-09-14 at 4 06 26 PM](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/83e7c1f4-0bd7-497d-8dc5-e4692d8139fc)

For more information, please see:
https://docs.securityonion.net/en/2.4/firewall.html#configuration
